ui/apps/ui/src/app/pages/guidelines-page/guideline-detail-page.component.ts

```diff
@@ -33,6 +36,11 @@ export class GuidelineDetailPageComponent implements OnInit {
     private _router: Router
   ) {}
 
+  parseCreators(jsonDeserialized: string | undefined): Partial<ICreators>[] {
+    if (jsonDeserialized) console.log(JSON.parse(jsonDeserialized));
```

Comment:
`if`s without brackets can lead to issues - I would add them.

Reply:
true

Reply:
done